app/services/api/webflow-api.ts

```diff
@@ -77,38 +77,40 @@ const workshopsOptions = webflowOptions<RawWorkshop>(WORKSHOPS)
 export const useWorkshops = () => useQuery(workshopsOptions)
 
 const scheduledEventsOptions = webflowOptions<RawScheduledEvent>(SCHEDULE)
-const scheduledEventsQueries = [
-  speakersOptions,
-  workshopsOptions,
-  recurringEventsOptions,
-  talksOptions,
-  scheduledEventsOptions,
-] as const
+
 export const prefetchScheduledEvents = async () => {
-  await Promise.all(
-    scheduledEventsQueries.map(async (query) => {
-      return queryClient.prefetchQuery(query)
-    }),
-  )
+  const prefetchQueryOptions = [
+    speakersOptions,
+    workshopsOptions,
+    recurringEventsOptions,
+    talksOptions,
+  ] as const
```

Comment:
I removed `scheduledEventsOptions` from this list because `refetch` appears to bypass the `enabled` flag.